azext_iot/central/models/v1_1_preview/destination.py

```diff
@@ -0,0 +1,30 @@
+# coding=utf-8
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+
+class Destination:
+    def __init__(self, destination: dict):
+        self.id = destination.get("id")
+        self.display_name = destination.get("displayName")
+        self.type = destination.get("type")
+        self.authorization = destination.get("authorization")
+        self.status = destination.get("status")
+        self.error = destination.get("errors")
+
+
+class WebhookDestination(Destination):
+    def __init__(self, destination: dict):
+        super().__init__(destination)
+        self.url = destination.get("url")
+        self.headerCustomizations = destination.get("headerCustomizations")
```

Comment:
nit: self.header_customizations
cli core helper methods automatically convert snake case to camel case for the output and we can keep code with python style

Reply:
fixed.